<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPropertiesWindow</class>
 <widget class="QDialog" name="CPropertiesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Schwellwertverfahren:&lt;/span&gt; &lt;br&gt;
H = Farbwert &lt;br&gt;
S = Farbs√§tigung&lt;br&gt;
V = Hellwert&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="Layout_HSV">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_H_MIN">
               <property name="text">
                <string>H_Min</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_H_MAX">
               <property name="text">
                <string>H_Max</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_S_MIN">
               <property name="text">
                <string>S_Min</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_S_MAX">
               <property name="text">
                <string>S_Max</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_V_MIN">
               <property name="text">
                <string>V_Min</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_V_MAX">
               <property name="text">
                <string>V_Max</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLCDNumber" name="lcd_H_MIN">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="value" stdset="0">
                <double>0.000000000000000</double>
               </property>
               <property name="intValue" stdset="0">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcd_H_MAX">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="value" stdset="0">
                <double>256.000000000000000</double>
               </property>
               <property name="intValue" stdset="0">
                <number>256</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcd_S_MIN">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcd_S_MAX">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="intValue" stdset="0">
                <number>256</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcd_V_MIN">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcd_V_MAX">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="intValue" stdset="0">
                <number>256</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <widget class="QSlider" name="slider_H_MIN">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_H_MAX">
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="value">
                <number>256</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_S_MIN">
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_S_MAX">
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="value">
                <number>256</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_V_MIN">
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_V_MAX">
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="value">
                <number>256</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="label_Erode">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Erode</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcd_Erode">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>70</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDial" name="dial_Erode">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>50</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="label_Dilate">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Dilate</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcd_Dilate">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>70</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDial" name="dial_Dilate">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>50</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Objektgr√∂√üe:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_BLUB_MIN">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="label_BLUB_MIN">
                 <property name="text">
                  <string>MIN </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcd_BLUB_MIN">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="slider_BLUB_MIN">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="singleStep">
                  <number>0</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_BLUB_MAX">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="label_BLUB_MAX">
                 <property name="text">
                  <string>MAX</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcd_BLUB_MAX">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="intValue" stdset="0">
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="slider_BLUB_MAX">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>47</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Originalbild:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="image_cam">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>320</width>
               <height>200</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Schwellwertverfahren:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="image_inRange">
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>320</width>
               <height>200</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item>
              <widget class="QPushButton" name="button_OK">
               <property name="text">
                <string>OK</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="button_cancel">
               <property name="text">
                <string>Abbrechen</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="button_reset">
               <property name="text">
                <string>Zur√ºcksetzen</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Erosionsverfahren:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="image_Erode">
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>320</width>
               <height>200</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Dilationsverfahren</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="image_Dilate">
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>320</width>
               <height>200</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Koordinaten:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>100</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dial_Dilate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_Dilate</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_Erode</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_Erode</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_H_MAX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_H_MAX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_H_MIN</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_H_MIN</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_S_MAX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_S_MAX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_S_MIN</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_S_MIN</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_V_MAX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_V_MAX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_V_MIN</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_V_MIN</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_BLUB_MAX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_BLUB_MAX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_BLUB_MIN</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_BLUB_MIN</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
